### PR TITLE
Receive messages but no interpolation yet

### PR DESCRIPTION
**Pull request by aftersomemath**
Everything to receive messages except for the interpolation.

I left the interpolation out for this PR because it's really not related to the rest of the message handling code that this PR mostly comprises of.

### CODE REVIEW

File: include/iarc7_motion/AccelerationPlanner.hpp [hunk 1]
```diff
@@ -0,0 +1,214 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Acceleration Planner
+//
+// Implements receiving twist arrays and the acceleration ramps
+//
+////////////////////////////////////////////////////////////////////////////
+
+#ifndef ACCLERATION_PLANNER_H
```
Comment (aftersomemath) [inline include/iarc7_motion/AccelerationPlanner.hpp]: I just remembered theres a mistake in a commit message too

File: include/iarc7_motion/AccelerationPlanner.hpp [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+#define ACCLERATION_PLANNER_H
+
+#include <ros/ros.h>
+#include <algorithm>
+#include <vector>
+#include "gtest/gtest_prod.h"
+#include "iarc7_msgs/TwistStampedArrayStamped.h"
+#include "geometry_msgs/TwistStamped.h"
+#include "QuadVelocityController.hpp"
```
Comment (amiller27) [inline include/iarc7_motion/AccelerationPlanner.hpp]: How are we organizing `#include`s?  I'd like to have some kind of system

Comment (amiller27) [inline include/iarc7_motion/AccelerationPlanner.hpp]: Probably separate package headers, project headers, and system headers and alphabetize within categories

Comment (aftersomemath) [inline include/iarc7_motion/AccelerationPlanner.hpp]: We discussed this at the beginning of the semester however I'm clearly not following it.

I believe it was.

Associated header
System includes, alphabetical
other includes, alphabetical

I'd like other includes to become
everything not messages, alphabetical
messagetype, alphabetical

Comment (amiller27) [inline include/iarc7_motion/AccelerationPlanner.hpp]: I like it

File: include/iarc7_motion/AccelerationPlanner.hpp [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        // Subscriber for uav_arm
+        ros::Subscriber velocity_targets_subsriber_;
+
+        TwistStampedArray velocity_targets_;
+    };
+
+    template<class T>
+    AccelerationPlanner<T>::AccelerationPlanner(ros::NodeHandle& nh, T& velocity_controller) :
+    nh_(nh),
+    velocity_targets_subsriber_(),
```
Comment (amiller27) [inline include/iarc7_motion/AccelerationPlanner.hpp]: Lol this is spelled wrong too

**Pull request merged**